src/review-todo.js

```diff
@@ -0,0 +1,212 @@
+let isDate = require("date-fns/isDate");
+
+function todo(todoName = "New Todo") {
+	let title = todoName,
+		description = "",
+		urgency = 0,
+		isDone = false,
+		dueDate = todoDate(),
+		scheduleDate = todoDate(),
+		tags = tagList();
+
+	let todoObj = Object.create(todo.proto);
+	Object.assign(todoObj, {
+		title,
+		description,
+		isDone,
+		urgency,
+		dueDate,
+		scheduleDate,
+		tags,
+	});
+
+	return todoObj;
+}
+
+todo.proto = {};
+
+function project(projectName = "New Project") {
+	let proto = Object.assign(project.proto, { isChild });
+	let projectObj = Object.create(proto);
+
+	let title = projectName,
+		description = "",
+		favorite = false,
+		urgency = 0,
+		isDone = false,
+		dueDate = todoDate(),
```

Comment:
Another benefit of a constructor approach is that using `new` makes it much clearer to a reader on this that what you're doing is here is assigning an object instead of value. Right now, it's ambiguous and I have to look up this function definition to understand what `todoDate` is.